src/main/java/org/ladocuploader/app/cli/SubmissionTransfer.java

```diff
@@ -202,9 +223,9 @@ private static String removeFileExtension(String filename) {
   }
 
   private static String formatBirthdate(Map<String, Object> inputData) {
-    String day = (String) inputData.get("birthDay");
-    String month = (String) inputData.get("birthMonth");
-    String year = (String) inputData.get("birthYear");
+    String day = (String) inputData.getOrDefault("birthDay", "");
```

Comment:
I am surprised that we need a default for birthday since it is a required field. Wondering if this allows partial applications to be submitted (if there is a split session issue for example).  🤔

Reply:
Yeah - unlikely it will be null, but just added it as a safeguard